packages/core/src/awsService/appBuilder/serverlessLand/main.ts

```diff
@@ -115,9 +115,10 @@ async function openReadmeFile(config: CreateServerlessLandWizardForm): Promise<v
             getLogger().warn('README.md file not found in the project directory')
             return
         }
+        await new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
why do we need a sleep? We also have a sleep util

Reply:
We need sleep to give time for README preview to load, so that it opens correctly. I have used `sleep` util now. Thanks!

Reply:
? Use `waitUntil` to check for the file to exist. sleep() is a poor user experience, and also fragile (some systems could be very slow temporarily, for various reasons).